### ISSUE TITLE
light models got bad mAP

### ISSUE THREAD

**Issue by luohao123** [duwt/OrienMask#14]
I am using exactly same backbone of YOLOX-s, but the AP is very low, but the mask and box seems normal. AP about 21.

Do u know why? Have a test with some light backbone models? Hope you can did more experiment on different backbone on the performance changes.

**Comment by duwt** [comment]
If the AP gap between box and mask is small but the performance is poor, the model convergence should be doubted.

**Comment by luohao123** [comment]
@duwt I haven't test mask AP yet, but box AP is low. I trained about 20000 iterations whole on 2 GPU card. I think it's enough. but the AP can not even better than 30. I don't know it's i am using yolox-s a small version backbone caused this or not. But YOLOX-s can achieve a 40 mAP on detection (even though it using anchor free and many augmentations).

But I think at least detection mAP should be 34 to be normal. what do u think?

**Comment by duwt** [comment]
The official YOLOv3 needs to be trained over 270 epochs on COCO. I guess 20000 iterations of training from scratch is not enough.

**Comment by luohao123** [comment]
@duwt sorry, it's 200000 iterations

**Comment by luohao123** [comment]
@duwt I am using yolox-s pretrained model. the AP biggest is only 21 for now. Do u think what could be the reason of this?
